tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleDependencies.java

```diff
@@ -262,10 +262,10 @@ public class ExampleDependencies {
     .put("//ldml/units/unitLength[@type=\"*\"]/unit[@type=\"*\"]/unitPattern[@count=\"*\"]", "//ldml/units/unitLength[@type=\"*\"]/compoundUnit[@type=\"*\"]/compoundUnitPattern")
     .put("//ldml/units/unitLength[@type=\"*\"]/unit[@type=\"*\"]/unitPattern[@count=\"*\"]", "//ldml/units/unitLength[@type=\"*\"]/compoundUnit[@type=\"*\"]/compoundUnitPattern1[@count=\"*\"]")
     .putAll("//ldml/personNames/sampleName[@item=\"*\"]/nameField[@type=\"*\"]",
-        "//ldml/personNames/personName[@length=\"*\"][@usage=\"*\"][@style=\"*\"][@order=\"*\"]/namePattern",
-        "//ldml/personNames/personName[@length=\"*\"][@style=\"*\"][@order=\"*\"]/namePattern")
+        "//ldml/personNames/personName[@order=\"*\"][@length=\"*\"][@usage=\"*\"][@formality=\"*\"]/namePattern",
+        "//ldml/personNames/personName[@order=\"*\"][@length=\"*\"][@style=\"*\"]/namePattern")
```

Comment:
@macchiati Thanks for the fixes, but shouldn't the second "style" here also be changed to "formality"? I can make that fix when I reconcile the conflicts.

Reply:
Fixed in 3rd commit